src/components/DialogFooter/DialogFooter.stories.tsx

```diff
@@ -0,0 +1,40 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import { DialogFooter, DialogFooterProps } from './DialogFooter';
+import { ButtonEmphasis, ButtonStyle } from '@components/Button';
+import { IconCheckMark } from '@foundation/Icon';
```

Comment:
Shouldn't we import from `@foundation/Icon/Generated`? https://github.com/Frontify/fondue/search?q=%40foundation%2FIcon%2FGenerated